### ISSUE TITLE
No original code, just an account for forked/cloned open source work

### ISSUE THREAD

**Issue by WMBranch** [CommerceGov/Policies-and-Guidance#7]
This might be covered in the issues and comments here already, but I'm copying it here now at Ian's request to keep the current questions and issues in one place.

Our shiny new Drupal-based website uses open source code. Some of that’s hosted on GitHub. Some of it is hosted by Drupal.org in their own git repository.

Because we can’t take a chance on having some code we need disappear, we’re going to fetch everything we use from the public, open source repos and keep it on machines we control. We’ll watch for updates and pull them for testing within our complete code base. We’ll keep our copies of the stuff we used in previous builds.

To that end, I went ahead and created a GitHub account that uses my .gov email address. I did it so that I would have a place to practice pulling content & noticing updates from GitHub. 

I never put any real code or real content there, and we don’t intend right now to put anything there. We're not in the business of open source development, for Drupal or anything else right now, and if that changes the policy is clear that we would go through the official USPTO and DOC accounts.

But to stay on the right side of the law, I just wanted to be sure I can do this kind of work (getting to know git and how best to interact/keep up with the open source community’s work that we depend on) this way.

In a nutshell: It has no govt content or code, and won’t, but I registered the GitHub account with a .gov email address. Is that cool?
